fedimint-client/src/module/init.rs

```diff
@@ -107,13 +117,15 @@ where
     C: ClientModuleInit,
 {
     federation_id: FederationId,
+    num_peers: NumPeers,
```

Comment:
future work: Now that we have this argument here we should check if we can remove the `NumPeersExt` impl for key sets etc.

Reply:
:+1:

Was never a fan of NumPeersExt on collections etc. Very surprising when reading code without LSP.

Reply:
I think the reason for it was that we didn't have the `Args` struct and didn't want to add yet another argument but reuse existing ones. Since that's not a concern anymore let's get rid of it.